api/src/storage/index.test.ts

```diff
@@ -0,0 +1,51 @@
+// import { StorageManager } from '@directus/drive';
```

Comment:
This should probably go into the the To-Do list as well 🤓

Reply:
Whoooops! Good one 🙂